Signal/src/ViewControllers/ConversationView/Cells/OWSAudioMessageView.m

```diff
@@ -247,8 +240,24 @@ - (void)createContents
     bottomLabel.textColor = [textColor colorWithAlphaComponent:0.85f];
     bottomLabel.lineBreakMode = NSLineBreakByTruncatingMiddle;
     bottomLabel.font = [OWSAudioMessageView labelFont];
+
+    UIStackView *labelsView = [UIStackView new];
+    labelsView.axis = UILayoutConstraintAxisVertical;
+    labelsView.spacing = [OWSAudioMessageView labelVSpacing];
+    [labelsView addArrangedSubview:topLabel];
+    [labelsView addArrangedSubview:audioProgressView];
     [labelsView addArrangedSubview:bottomLabel];
 
+    // Ensure the "audio progress" and "play button" are v-center-aligned using a container.
+    UIView *labelsContainerView = [UIView containerView];
```

Comment:
No biggie, but I don't see why we'd need this containerView to v-align the audioProgressView to the playPauseButton.